### ISSUE TITLE
Multiple operands return null while options return empty array

### ISSUE THREAD

**Issue by tflori** [getopt-php/getopt-php#119]
snippet from a psy shell:
```
>>> GetOpt\Operand::create('test', GetOpt\Operand::MULTIPLE)->getValue()
=> null
>>> GetOpt\Option::create('t', 'test', GetOpt\GetOpt::MULTIPLE_ARGUMENT)->getValue()
=> []
```

I would say it is a bug but maybe someone has a different meaning?

**Comment by tflori** [comment]
In #120 this is solved. Should we back port it to 3.1?

**Comment by tflori** [comment]
solved with #126
